communities/aston-css/community-impact-proposal.md

```diff
@@ -0,0 +1,43 @@
+
+Along the same kind of lines, and based on suggestions from Joe and the actions of others, I'd like to write a Code of Conduct for the society. I mentioned about about the culture of the society, and how it can sometimes lead to offence or alienation of some of our members/outsiders, and I think having a clear CoC could resolve some of these issues. It will also act as a more official way of dealing with conflict, something which is a bit of a grey area at the moment.
+
+I plan to create aston-css/code-of-conduct repo and either a repo for the manifesto (like aston-css/manifesto) or just use our [new website](https://astoncss.com/) and tweak the words to be more specific, like a manifesto. I hope to get both of these done pretty soon, maybe within the next month or so.
+
+### Programme of Events
+
+Continuing with the plan of running regular events, I'd like to come up with a well thought-out plan of talks and workshops. The aim of these will be to draw in interest from more people, and to accomplish this the events will be marketed as helping members achieve something (eg, building a personal site on GitHub pages). I think events are more likely to appeal to people if they know what they're going to get out of them - an actual _thing_ on top of new knowledge.
+
+If we plan out the entire term of events (at least a rough plan) in advance, we can do more ambitious things, and build on top of previous sessions to cover more advanced topics. So we'd lay out the foundations at the start of the year - things like Git, using the terminal, etc.
+
+As mentioned before, we have many members to take advantage of in terms of running talks and workshops. I said that we could have a CFP within our community, and I think this would work really well along with a bit of a nag to specific members!
+
+The summer will act as the planning phase for this, and the entirety of term 1 will be the execution phase.
+
+## Planning for the Future
+
+Certainly with the documentation, this is going to have a lasting effect. The manifesto acts as the definition of what the society is all about, and should help drive decisions that this committee and future committees make. It's also important that this is an evolving document which can be edited and commented on.
+
+I don't think that the docs necessarily have a measurable impact, so it's difficult to work out how successful they are. Perhaps a good idea would be for the committee to review the docs twice a year - once with the new incoming committee, and then once again around mid-term. Members should also be actively encouraged to review them.
```

Comment:
Engagement with the docs would be interesting, perhaps hooking them up to google analytics.